content/en/blog/_posts/2023-03-10-image-registry-change.md

```diff
@@ -33,83 +33,24 @@ registry](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-g
-
-For more details on this change, check out [registry.k8s.io: faster, cheaper and Generally Available
-(GA)](/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/).
-
-## Why is a redirect being put in place?
-
-The project switched to [registry.k8s.io last year with the 1.25
-release](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/); however, most of
-the image pull traffic is still directed at the old endpoint k8s.gcr.io. This has not been
-sustainable for us as a project, as it is not utilizing the resources that have been donated to the
-project from other providers, and we are in the danger of running out of funds due to the cost of
-serving this traffic.
-
-A redirect will enable the project to take advantage of these new resources, significantly reducing
-our egress bandwidth costs. We only expect this change to impact a small subset of users running in
-restricted environments or using very old clients that do not respect redirects properly.
-
-## What images will be impacted?
-
-**ALL** images on k8s.gcr.io will be impacted by this change. k8s.gcr.io hosts many images beyond
-Kubernetes releases. A large number of Kubernetes subprojects host their images there as well. Some
-examples include the `dns/k8s-dns-node-cache`, `ingress-nginx/controller`, and
-`node-problem-detector/node-problem-detector` images.
-
-## What will happen to k8s.gcr.io?
-
-Separate from the the redirect, k8s.gcr.io will be frozen [and will not be updated with new images
-after April 3rd, 2023](https://kubernetes.io/blog/2023/02/06/k8s-gcr-io-freeze-announcement/). `k8s.gcr.io` 
-will not get any new releases, patches, or security updates. It will continue to remain available to
-help people migrate, but it **WILL** be phased out entirely in the future.
-
-## I run in a restricted environment. What should I do?
-
-For impacted users that run in a restricted environment, the best option is to copy over the
-required images to a private registry or configure a pull-through cache in their registry.
-
-There are several tools to copy images between registries;
-[crane](https://github.com/google/go-containerregistry/blob/main/cmd/crane/doc/crane_copy.md) is one
-of those tools, and images can be copied to a private registry by using `crane copy SRC DST`. There
-are also vendor-specific tools, like e.g. Google’s
-[gcrane](https://cloud.google.com/container-registry/docs/migrate-external-containers#copy), that
-perform a similar function but are streamlined for their platform.
-
-## How can I check registry.k8s.io is accessible from my cluster?
+## How can I check if I am impacted?
 
 To test connectivity to registry.k8s.io and being able to pull images from there, here is a sample
 command that can be executed in the namespace of your choosing:
 
 ```
-kubectl run hello-world --tty --rm -i --image=registry.k8s.io/busybox:latest sh
+kubectl run hello-world -ti --rm --image=registry.k8s.io/busybox:latest --restart=Never -- date
```

Comment:
nit: this change doesn't match the PR title

Reply:
+1 to this change however, seems like a worthwhile improvement.

Reply:
title updated :)